server/public/Makefile

```diff
@@ -0,0 +1,14 @@
+test-public: gotestsum
+	$(GOBIN)/gotestsum ./public/... -- $(GOFLAGS)
+
+## Generates mock golang interfaces for testing
```

Comment:
Can we just include this in `server/Makefile`? While we're shipping this submodule discretely, I don't think we need to encumber it with a distinct workflow.

Reply:
1/5 it is a separate submodule that deserves its own `Makefile`. The main `Makefile` is already over 800 lines long.